daisy_workflows/image_build/debian/debian_10_fai.wf.json

```diff
@@ -28,7 +28,6 @@
           "build_date": "${build_date}",
           "debian_cloud_images_version": "${debian_cloud_images_version}",
           "debian_version": "${debian_version}",
```

Comment:
it's fine for this to be parameterized, but code the default into the base rather than repeating for each invocation